### DISCUSSION TITLE
Construction photo timelapse. Each day same time.

### DISCUSSION THREAD

**Discussion by Mattie112** [gopro/labs#584] [Q&A]
Hi,

For a construction project I'd like to take a picture every x minutes between 08:00 and 18:00 for example. I installed the labs firmware but I cannot get it to work like this. I just want pictures, I will manually create a timelapse for this. My camera is a Hero 10. With firmware LABS_HERO10_01_50_70.

I have tried:
https://gopro.github.io/labs/control/custom/
But this has no "between 08:00 and 18:00" option, only a "delay" option.

I have looked at:
https://gopro.github.io/labs/control/longtimelapse/
But I just want it to take more pictures then this.

This seems what I need:
https://gopro.github.io/labs/control/construction/
But it is marked deprecated. Still tried it, 0 pictures taken even though this morning it said the first picture would be taken in 30minutes before I left for work.

I tried using the app:
Used the "photo-tl"mode but this starts a "video" timelapse. And I want to save battery so I want it to turn off in between.

I did find some suggestions here but I could not find my specific case.

Does anyone know know what settings to use?

**Comment by Mattie112** [comment]
For example with the "construction" QR (I did replace the battery just to be sure around 07:41 and then rescanned the QR code)

```
[2023-10-02 07:41:20] Read QR:mPdP>07:59<17:00!309SQ~!08:00S!1R 
[2023-10-02 07:41:22] false 07:41 > 07:59:>07:59<17:00!309SQ~!08:00S!1R 
[2023-10-02 07:41:25] Shutdown and Sleep:997 seconds 
[2023-10-02 07:41:26] shutdown:rtc wake scheduled 
[2023-10-02 07:58:05] Labs:Main Start 
[2023-10-02 07:58:06] Labs:Initialize 
[2023-10-02 07:58:06] Woke with Alarm:[08:00:00]!S!1R 
[2023-10-02 07:58:08] Woke with Command:[08:00:00]!S!1R 
[2023-10-02 07:58:09] Shutdown and Sleep:92 seconds 
[2023-10-02 07:58:13] shutdown:rtc wake scheduled 
[2023-10-02 07:59:47] Labs:Main Start 
[2023-10-02 07:59:48] Labs:Initialize 
[2023-10-02 07:59:48] Woke with Alarm:[08:00:00]!S!1R 
[2023-10-02 07:59:50] Woke with Command:[08:00:00]!S!1R 
[2023-10-02 07:59:50] Start:10 seconds 
[2023-10-02 08:00:01] Timer wake command:!S!1R 
[2023-10-02 19:23:42] Labs:Main Start 
[2023-10-02 19:23:42] Labs:Initialize 
```

**Comment by dnewman-gpsw** [comment]
You don't mention your camera model or firmware version.  This feature should work okay on all models, but I can try if you share those details.

The construction script is not recommended, only as it not as reliable as just shooting timelapse continuously under external power.  It normally works for about a week, before requiring user intervention.  As construction projects are typically longer, it generally easier to use an built in camera timelapse feature, and just delete the extra photos in post.  If you really want daytime only photos, then use an external AC power timer and use boot commands, which is much more reliable.

**Comment by Mattie112** [comment]
Ah sorry that would have been a good idea :)

I have a Hero 10 camera. I have just downloaded the firmware LABS_HERO10_01_50_70

I understand that that would be an option but that is more of a hassle to remove all the dark photos at a later time. But if that is the only thing that works hm...

The idea of a power-timer is quite fun I do have some options for that but it is strange to me that it doesn't work with this firmware.

**Reply by dnewman-gpsw** [reply]
The issue is you asking a consumer camera to boot say 100 times a day, seven days a week, and using an experimental feature. The camera's only schedule capture only wakes once per day, which is 100X more reliable.  

Most reliable to least (percentage is the chance each capture will succeed.)
1) 100% - Power the camera from wall, with USP, so no issues with power. Record with photo timelapse continuously.
2) 99.9999% - Same power source, using an external timer 
![image](https://github.com/gopro/labs/assets/10856084/5adca33b-5cbb-4356-abec-746f408bb25f)
this have a boot command that start capturing photo timelapse whenever the camera is on. The system is not affected if the camera forgets time. 
3) 99.9% Use a camera with external power, and use the camera's internal timer to start a daily photo timelapse.  This risks the camera RTC time being forgotten, which does happen (particularly without internal battery.)
4) 99% Wake the camera for ever photo.

**Reply by Mattie112** [reply]
Allright thanks for the suggestions.

I thought that the camera built-in timelapse would always create a "video" instead of just taking regular photos. I still think it is a waste to "record" through the night so I might try it with option 2 or 3. I can power the camera with but is is a construction project so the power might to out a few times.

**Reply by dnewman-gpsw** [reply]
Time-lapse in camera can be either photo or video, just edit the preset to select your preference.

**Reply by Mattie112** [reply]
Allright, I will now try it powered (no battery, not sure how hot it will get will try that this weekend) with the following config: https://gopro.github.io/labs/control/set/?cmd=!08:00NmTPp.300fL0oB1D4S1!S!18:00E!R

I have a good quick-charge charger that should supply 5v 3A so that should work :)

**Reply by dnewman-gpsw** [reply]
Always experiment with a short term test first
